### PR TITLE
Adding 2 new sample projects in the gallary

### PR DESCRIPTION
**Pull request by shizn**
Blob storage project 
Module testing loop project

### CODE REVIEW

File: sample-gallery.json [hunk 1]
```diff
@@ -40,6 +40,40 @@
...
+              }
+            ],
+            "featured": false
+        },
+        {
+            "name": "Module Testing Loop",
+            "project_name": "e2e_testing_loop",
+            "description": "A sample solution for testing an Azure IoT Edge pipeline with Azure Functions and Application Insights",
+            "url": "Azure-Samples/iotedge-end2end-messageflow",
+            "issuesUrl": "https://github.com/Azure-Samples/Custom-vision-service-iot-edge-raspberry-pi/issues",
```
Comment (xinyiz1226) [inline sample-gallery.json]: is this the correct link?

File: sample-gallery.json [hunk 2]
```diff
@@ -40,6 +40,40 @@
...
             ],
             "featured": false
+        },
+        {
+            "name": "Blob Storage",
+            "project_name": "blob_storage_sample",
+            "description": "Simple sample showing how to use local blob storage with Azure IoT Edge.",
+            "url": "Azure-Samples/azure-iotedge-blobstorage-sample",
+            "issuesUrl": "https://github.com/Azure-Samples/azure-iotedge-blobstorage-sample/issues",
+            "image": "https://github.com/Azure-Samples/vscode-azure-iot-edge-sample-gallery/raw/master/images/blob-storage.png",
```
Comment (adashen) [inline sample-gallery.json:50]: image has not been uploaded right?

File: sample-gallery.json [hunk 3]
```diff
@@ -40,6 +40,40 @@
...
+            ],
+            "featured": false
+        },
+        {
+            "name": "Module Testing Loop",
+            "project_name": "e2e_testing_loop",
+            "description": "A sample solution for testing an Azure IoT Edge pipeline with Azure Functions and Application Insights",
+            "url": "Azure-Samples/iotedge-end2end-messageflow",
+            "issuesUrl": "https://github.com/Azure-Samples/Custom-vision-service-iot-edge-raspberry-pi/issues",
+            "image": "https://github.com/Azure-Samples/vscode-azure-iot-edge-sample-gallery/raw/master/images/end-to-end-testing-solution.png",
```
Comment (adashen) [inline sample-gallery.json:67]: image need to be uploaded

Comment (shizn) [reply sample-gallery.json:67]: images will be there after get merged.

**Pull request merged**